atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleOverflowQueryFactory.java

```diff
@@ -52,8 +49,10 @@ public boolean hasOverflowValues() {
         for (OverflowValue overflowId : overflowIds) {
             oraRows.add(new Object[] { null, null, overflowId.id });
         }
-        ArrayHandler arg = jdbcHandler.createStructArray("PT_MET_CELL_TS", "PT_MET_CELL_TS_TABLE", oraRows);
-        switch (migrationState) {
+        ArrayHandler arg = config.jdbcHandler().createStructArray(
+                config.shared().tablePrefix() + "CELL_TS",
```

Comment:
fix this in https://github.com/palantir/atlasdb/pull/540 (it ends up mattering because hard-coded internal code uses upper case and oracle is case-sensitive here)